src/content/docs/cypher/syntax.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Syntax
+description: Basics of the Cypher query language syntax
+---
+
+In this page, we list the syntactic features of Cypher as implemented in Kùzu. As described in the
+[overview](/cypher) page, Cypher is a declarative graph query language, and Kùzu's implementation is
+based on [openCypher](https://opencypher.org/resources/).
+
+## Parsing
+
+The query parser looks for an input `STRING` that consists of ASCII or unicode characters. The parser is case-insensitive
+and ignores leading and trailing whitespaces. You can use characters from non-English languages.
+To use special characters or unicode characters as identifiers, you can escape them by encapsulating
+them in backticks \`, such as \``Ψ`\`.
+
+```cypher
+// Create a node table of books in German
+CREATE NODE TABLE `B\u00fccher` (title STRING, price INT64, PRIMARY KEY (title))
+CREATE (n:`B\u00fccher` {title: 'Der Thron der Sieben Königreiche'}) SET n.price = 20
+// Query using the unicode representation of the table name
+MATCH (n:Bücher) RETURN label(n)
+```
+```
+┌─────────┐
+│ Bücher  │
+│ STRING  │
+├─────────┤
+│ Bücher  │
+└─────────┘
+```
+
+Breaking a query into multiple lines is allowed (and recommended for readability reasons), and the
+parser ignores leading and trailing whitespaces. You can explicitly mark the end of a query with a semicolon `;`.
+
+```cypher
+MATCH (a:Person)
+WHERE a.age < 30
+RETURN a.*;
+```
+
+## Clauses
+
+A Cypher query may contain one or more clauses and their associated subclauses, and can span multiple lines.
+The end of a statement is marked with a semicolon `;`, and the query parser looks for this symbol
+to know when a statement is complete.
+
+Examples of clauses include:
+- `MATCH`: Find patterns in the graph
+- `RETURN`: Specify what subset of the matched data to return
+
+Examples of subclauses (that must reside under a clause) include:
+- `WHERE`: Filter the results of a `MATCH` clause
+- `LIMIT`: Limit the number of results returned by a query
+
+## Comments
+
+Comments are for humans to read and document their code, and are ignored by the
+query parser.
+- Single line comments begin with a double slash (`//`) and continue up until the end of the
+line. They can be placed at the beginning, in the middle, or at the end of a query.
+- Multi-line comments begins with a slash and asterisk (`/*`) and continues until it ends with an
+asterisk and a slash (`*/`). They can be useful for comments that are too long for one line.
+
+Some examples are below.
+
+```cypher
+// Whole-line comment before a query
```

Comment:
I recall we had an issue here on the CLI end. Can u try and open an issue if necessary? If we do have a bug, let's not document this example.

Reply:
I'm waiting for the bug to be resolved before this is merged :). It currently works in Python API and Kùzu Explorer, I just tested there. But for CLI, it isn't working yet. Will check if there's an issue, and if not, will post one.